test/Splitter.t.sol

```diff
@@ -141,23 +140,23 @@ contract SplitterTest is DssTest {
             bump:                5707 * RAD,
             hop:                 30 minutes,
             burn:                70 * WAD / 100,
-            usdsJoin:            DAI_JOIN,
+            usdsJoin:            USDS_JOIN,
             splitterChainlogKey: "MCD_FLAP_SPLIT",
-            prevMomChainlogKey:  "FLAPPER_MOM",
+            prevMomChainlogKey:  "SPLITTER_MOM",
             momChainlogKey:      "SPLITTER_MOM"
         });
         FlapperUniV2Config memory flapperCfg = FlapperUniV2Config({
             want:            WAD * 97 / 100,
             pip:             address(medianizer),
-            pair:            UNIV2_DAI_MKR_PAIR,
-            usds:            DAI,
+            pair:            UNIV2_SKY_USDS_PAIR,
+            usds:            USDS,
             splitter:        address(splitter),
             prevChainlogKey: bytes32(0),
             chainlogKey:     "MCD_FLAP_BURN"
         });
         FarmConfig memory farmCfg = FarmConfig({
             splitter:        address(splitter),
-            usdsJoin:        DAI_JOIN,
+            usdsJoin:        USDS_JOIN,
```

Comment:
I think we can also now change `FlapperInit.setFarm` to use the `usds()` view function instead of `dai()` (and remove the corresponding TODO).

Reply:
Done.